### PR TITLE
Text updates for front page.

### PR DESCRIPTION
**Pull request by geertj**

### CODE REVIEW

File: site/_includes/solutions.html [hunk 1]
```diff
@@ -6,11 +6,13 @@
...
 
       <div class="solution-content">
-        <h3 class="solution-title">Discovery</h3>
+        <h3 class="solution-title">Continous Discovery</h3>
         <div class="solution-text">
-          ManageIQ connects to the APIs of your virtualization, container,
-          network and storage management systems and continuously discovers
-          your resources and maps their relationships.
+          Connect ManageIQ to your virtualization, container, network, and
+          storage management systems, where it will discover inventory, map
```
Comment (Fryguy) [inline site/_includes/solutions.html]: middleware?

File: site/_includes/solutions.html [hunk 2]
```diff
@@ -44,9 +46,10 @@ <h3 class="solution-title">Self-service</h3>
       <div class="solution-content">
         <h3 class="solution-title">Compliance</h3>
         <div class="solution-text">
-          ManageIQ can scan the contents of your VMs, hosts and containers,
-          without the help of an agent. Using this data you can define
-          advanced compliance policies and take action.
+          Scan the contents of your VMs, hosts, and containers, and combine
+          with auto discovery data to create advanced security and compliance
+          policies. Content scanning works without the help of an agent and
+          therefore works for any VM including foreign and un-cooperative ones.
```
Comment (Fryguy) [inline site/_includes/solutions.html]: Should we mention powered-off vms?

File: site/_includes/solutions.html [hunk 3]
```diff
@@ -57,9 +60,10 @@ <h3 class="solution-title">Compliance</h3>
       <div class="solution-content">
         <h3 class="solution-title">Optimization</h3>
         <div class="solution-text">
-          ManageIQ captures metrics on your VMs, containers, and more.
-          It can find unused systems, make right-sizing recommendations,
-          and do capacity planning.
+          Capture metrics for your VMs, containers, and hosts to understand
+          currently utilization and normal operating ranges. Then use this data
```
Comment (Fryguy) [inline site/_includes/solutions.html]: `currently` => `current`?  Not sure, but currently reads funny

**Comment by Fryguy** [comment]
A couple of comments...I really like this new copy.

File: site/_includes/solutions.html (hunk 3 above)
Comment (geertj) [inline site/_includes/solutions.html]: Typo indeed, will update.

File: site/_includes/solutions.html (hunk 2 above)
Comment (geertj) [inline site/_includes/solutions.html]: I tried really hard to get it down to 3 lines on the front page ;)

**Pull request merged**